analysis_options.yaml

```diff
@@ -4,3 +4,7 @@ include: package:flutter_lints/flutter.yaml
 # https://dart.dev/guides/language/analysis-options
 linter:
   rules:
+
+analyzer:
+  exclude:
+    - test/**.mocks.dart
```

Comment:
What was analysis's problem with the mock file? I already have some, but haven't had a failure yet.

Reply:
Oh, I see in the Action log `The '?' is unnecessary because 'Null' is nullable without it • test/fitness_equipment_test.mocks.dart:153:18 • unnecessary_question_mark`. Oh well. We can exclude the mocks!